misc/conda/make_conda_env.sh

```diff
@@ -233,12 +255,10 @@ echo "  conda activate $ENVNM"
 echo "The environment can be deactivated with the command"
 echo "  conda deactivate"
 echo
-echo "Jax installed without GPU support. To avoid warning messages,"
-echo "add the following to your .bashrc or .bash_aliases file"
-echo "  export JAX_PLATFORM_NAME=cpu"
-echo "To include GPU support, reinstall the astra-toolbox conda"
-echo "package on a host with GPUs, and see the instructions at"
-echo "   https://github.com/google/jax#pip-installation-gpu-cuda"
+echo "Jax installed without GPU support. To include GPU support,"
+echo "reinstall the astra-toolbox conda package on a host with GPUs,"
+echo "and see the instructions at"
+echo "   https://jax.readthedocs.io/en/latest/installation.html#nvidia-gpu"
```

Comment:
I don't understand why both Jax and astra-toolbox are mentioned here.

Reply:
If I recall correctly (see [this issue](https://github.com/astra-toolbox/astra-toolbox/issues/327)), astra is installed without GPU support if installed on a host without GPUs, so the `astra-toolbox` reinstall is necessary to include the required CUDA dependencies.